apps/wallet-mobile/src/features/Receive/common/ShareQRCodeCard/ShareQRCodeCard.stories.tsx

```diff
@@ -7,7 +8,9 @@ import {ShareQRCodeCard} from './ShareQRCodeCard'
 
 storiesOf('Receive ShareQRCodeCard', module)
   .addDecorator((story) => <View style={styles.container}>{story()}</View>)
-  .add('default', () => <ShareQRCodeCard address={mocks.address} />)
+  .add('with content', () => (
+    <ShareQRCodeCard content={mocks.address} onLongPress={action('onLongPress')} title="Title" />
```

Comment:
most components in common are actually useCases of just one screen, it needs a second refactor later